ci-operator/step-registry/ipi/deprovision/nutanix/preload-image/ipi-deprovision-nutanix-preload-image-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: ipi-deprovision-nutanix-preload-image
+  from: tools
```

Comment:
I saw the step only need `curl`, very common. I think you can replace the image with:
```
  from_image:
    namespace: origin
    name: centos
    tag: '8'
```
To avoid user have to define the `tools` image in ci yaml file.